test/fixtures/this-methods.html

```diff
@@ -9,6 +9,16 @@
       doThing: function() {
         var imagePath = this.resolvePath('image.png');
         console.log(imagePath);
+        this.imagePath = imagePath;
+        this.deep.prop = 2 * 2;
+        this.deep[prop].of.another.place[toGo + '1'] = 2 * 2;
+        // Don't upgrade expressions inside inner functions
```

Comment:
I.. probably will actually. Some code that I'm upgrading uses it a decent amount. Added a TODO.